pom.xml

```diff
@@ -256,6 +256,16 @@
                 <artifactId>junit-jupiter-api</artifactId>
                 <version>${junit.jupiter.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>3.11.0</version>
```

Comment:
we have mockito version defined in common pom, should we use that instead
`<mockito.version>4.6.1</mockito.version>`